scripts/make_stm32_module.py

```diff
@@ -45,17 +48,24 @@ def module(make_serie):
 
     #  for CMSIS files
     os.mkdir(os.path.join(make_path, "soc"))
-    for repo_file in os.listdir(os.path.join(make_repo, "Drivers/CMSIS/Device/ST",
-                                             make_SERIExx, "Include")):
-        repo_src = os.path.join(make_repo, "Drivers/CMSIS/Device/ST",
-                                make_SERIExx, "Include", repo_file)
+    make_Include = os.path.join(make_repo, "Drivers/CMSIS/Device/ST", make_SERIExx, "Include")
+    for repo_file in os.listdir(make_Include):
+        repo_src = os.path.join(make_Include, repo_file)
         repo_dst = os.path.join(make_path, "soc", repo_file)
         os.rename(repo_src, repo_dst)
 
-    for repo_file in os.listdir(os.path.join(make_repo, "Drivers/CMSIS/Device/ST",
-                                             make_SERIExx, "Source/Templates")):
-        repo_src = os.path.join(make_repo, "Drivers/CMSIS/Device/ST",
-                                make_SERIExx, "Source/Templates", repo_file)
+    for repo_file in os.listdir(
+        os.path.join(
+            make_repo, "Drivers/CMSIS/Device/ST", make_SERIExx, "Source/Templates"
+        )
+    ):
+        repo_src = os.path.join(
+            make_repo,
+            "Drivers/CMSIS/Device/ST",
+            make_SERIExx,
+            "Source/Templates",
```

Comment:
if using `join`, all path portions should an individual argument, `Source/Templates` --> `"Source", "Templates"`. I'd also suggest to take a look at `pathlib` for a more modern path handling.

Reply:
Done